profiles/epub.md

```diff
@@ -1,12 +1,33 @@
 # EPUB Profile
 
-While EPUB 2.x and 3.x can mostly be mapped directly to the Readium Web Publication Manifest, a number of metadata, collection roles and properties are still very specific to EPUB and not fully covered by the core specification.
+**Editors:**
 
-Thanks to the various extension points in place, this document defines a number of new collection roles and properties that are for the most part exclusive to EPUB.
+* Hadrien Gardeur ([De Marque](http://www.demarque.com))
```

Comment:
There is a question to solve: should each profile get specific editors or should we list them only in the core model = README? I vote for the latter.

Reply:
I would vote for the opposite as we could have specialized profiles and/or context documents. I don't see a valid reason for a top down approach for something modular.